pkg/inframanager/config/config.go

```diff
@@ -47,6 +47,7 @@ func ReadConfig(conf *Configuration, cfgFileName string) {
 	viper.SetDefault("Infrap4dGnmiServer.client-key", types.ManagerDefaultClientKey)
 	viper.SetDefault("Infrap4dGnmiServer.ca-cert", types.ManagerDefaultCACert)
 
+	viper.SetDefault("InfraManager.Addr", "localhost:50002")
```

Comment:
Is this the effective default or the one in the types.go?

Reply:
Both are same. The types.go file's address is the default value for the variable where the viper's default is the default value for the configuration file.

Reply:
So, can we remove following setting
DefaultInfraManagerAddr     = "127.0.0.1"
from types.go?